languages/go/exercises/concept/conditionals/.docs/instructions.md

```diff
@@ -0,0 +1,32 @@
+# Instructions
```

Comment:
A general comment on the `instructions`: Have a look at the instructions of other existing concept exercises (e.g. in C# folder). A story is very good and then it is good to go into a small section explaining each function.